docs/FeatureManagement/Samples/TargetingFilter.sample.json

```diff
@@ -0,0 +1,60 @@
+{
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "ComplexTargeting",
```

Comment:
Let's name feature flags consistently.

How about ComplexTargetingFeatureFlag?

Reply:
Started a thread on this above. Let's talk about it there https://github.com/Azure/AppConfiguration/pull/890#discussion_r1540636226.